Makefile

```diff
@@ -44,16 +51,19 @@ update_plugin_docs::
 	./scripts/update_plugin_docs.sh azure-native ${PLUGIN_VERSION_AZURE_NATIVE}
 	./scripts/update_plugin_docs.sh awsx ${PLUGIN_VERSION_AWSX}
 
-install::
-	${GO} install ./cmd/...
+.PHONY: install
+install: install_pulumi-language-yaml install_pulumi-converter-yaml
+
+# Install a binary onto GOPATH
+.PHONY: install_%
+install_%: bin/%
+	cp $< $(or $(shell ${GO} env GOBIN),$(shell ${GO} env GOROOT)/bin)/$*
+
 
 clean::
 	rm -f ./bin/*
 	rm -f pkg/pulumiyaml/testing/test/testdata/{aws,azure-native,azure,kubernetes,random,eks,aws-native,docker}.json
 
-ensure::
-	${GO} mod download
```

Comment:
Is this not necessary? I guess `go build` does this automatically?

Reply:
Yep.